src/main/java/com/microexam/classapi/repositories/ClassRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.microexam.classapi.repositories;
+
+import com.microexam.classapi.entities.Class;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ClassRepository extends CrudRepository<Class, Integer> {
```

Comment:
Might be better to switch to JPARepository(exteneds CrudRepository) as it allows for more functionality and in the future we will want to do pagination  on 'Classes'

Reply:
Ok, I will do that